### PR TITLE
Nrh 180  sidebar blurb

### PR DESCRIPTION
**Pull request by JeremyGibson**
#### What's this PR do?
Add the ability to create sidebar blurbs. Currently the supported blocks are RichText and Images.  Once #116 is merged we can add in benefit levels.

#### How should this be manually tested?

1. Go to any page, and open up the page editor.
2. You will note a new tab `Sidebar`
3. Open `Sidebar` and begin adding elements
4. Reorder as you wish
5. Close the editor and save the page
6. Note the sidebar is displayed as constructed. 

#### Do any changes need to be made before deployment to staging? production? (adding environment variables, for example)?
No.

### CODE REVIEW

**Comment by michael-caktus** [comment]
This rocks, I love this feature.

**Comment by michael-caktus** [comment]
Oddly, the images are saving, but from `MediaImage.create_from_request` I'm getting `IntegrityError: null value in column "height" violates not-null constraint`

**Comment by michael-caktus** [comment]
Also, one thing we should sort out now is the props violation in the Image block-- 
Screen Shot 2021-08-13 at 4 06 41 PM

We were probably a bit too eager to make that required, since we're allowing the data-url to be used in edit mode, pre-save.

**Review by michael-caktus — CHANGES_REQUESTED** [review]
See comments in conversation section.

**Comment by JeremyGibson** [comment]
> Oddly, the images are saving, but from `MediaImage.create_from_request` I'm getting `IntegrityError: null value in column "height" violates not-null constraint`

Hmmm. That is odd. I saw that in tests when I wasn't passing a good image object to the constructor.

File: spa/src/components/donationPage/DonationPage.js [hunk 1]
```diff
@@ -138,9 +138,10 @@ export default DonationPage;
 
 function getInitialFrequency(page) {
   const frequencyElement = page?.elements?.find((el) => el.type === 'DFrequency');
-  if (frequencyElement) {
+  if (frequencyElement.content) {
+    debugger;
```
Comment (michael-caktus) [inline spa/src/components/donationPage/DonationPage.js:142]: 👆🚨

**Pull request merged**